### PR TITLE
Add compatibility for Alpine Linux

### PR DESCRIPTION
**Pull request by haqa**
#### Pull Request (PR) description
Enable management of Lets Encrypt on Alpine Linux

### CODE REVIEW

**Comment by ekohl** [comment]
Normally these already exist. Does Alpine not create these as part of the base system?

**Comment by haqa** [comment]
No, while some base Alpine images have /usr/local not one I've seen so far has /usr/local/sbin by default. And if they do exist these are essentially a no-op.

File: manifests/init.pp [hunk 1]
```diff
@@ -115,12 +115,23 @@
...
+  }
+
   # Used in letsencrypt::certonly Exec["letsencrypt certonly ${title}"]
   file { '/usr/local/sbin/letsencrypt-domain-validation':
     ensure => file,
     owner  => 'root',
     group  => 'root',
     mode   => '0500',
     source => "puppet:///modules/${module_name}/domain-validation.sh",
+    require=> File['/usr/local/sbin'],
```
Comment (ekohl) [inline manifests/init.pp]: This hasn't been needed since Puppet 2.7 and file autorequires parent directoryies.

**Comment by ekohl** [comment]
I dislike managing these files since they can conflict with other modules.

**Comment by ekohl** [comment]
Closing for inactivity.

**Pull request closed without merging**